lib/haml_lint/lint.rb

```diff
@@ -1,3 +1,5 @@
+require 'haml_lint/severity'
```

Comment:
🤔 I think I had worked on adding a `Severity` class the first time I tried to make the `ProgressReporter` and had it here from that. I'll check if it's necessary and remove it if it proves to not be needed.